payouts.go

```diff
@@ -0,0 +1,95 @@
+package goshopify
+
+import (
+	"fmt"
+
+	"github.com/shopspring/decimal"
+)
+
+const payoutsBasePath = "shopify_payments/payouts"
+
+// PayoutService is an interface for interfacing with the payout endpoints of
+// the Shopify API.
+// See: https://shopify.dev/docs/api/admin-rest/2023-01/resources/payout
```

Comment:
this is missing an "s", should be 
https://shopify.dev/docs/api/admin-rest/2023-01/resources/payouts